pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -91,6 +104,24 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
     },
   });
 
+  console.log(methods.watch());
+  const [isInitTasks, setInitTasks] = React.useState(false);
+
+  const { register, getValues, setValue, handleSubmit, watch, reset } = methods;
+
+  const {
+    mode,
+    targets,
+    sources,
+    withKnown,
+    includedPackages,
+    excludedPackages,
+    customRulesFiles,
+    excludedRulesTags,
+    createdTasks,
```

Comment:
`createdTasks` are only used in this file and we can pass the task Id to `UploadBinary` which always handles a single application case anyway.